### ISSUE TITLE
How to check number of gradients collected by master node from python interface

### ISSUE THREAD

**Issue by zazzydistsgd** [apache/incubator-mxnet#6560]
Hi,

In distributed synchronous training (when setting `kv_store=dist_sync`) I want to keep checking how many works' gradients are currently aggregated by the `kvstore`, which is quite similar to the `ConditionalAccumulator.num_accumulated` in TensorFlow (https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/data_flow_ops.py#L1081). 
In particular, for example, if there are n workers in a distributed cluster, for each training iteration, I want to see the number of gradients gathered by the master (or parameter server) increasing from 0 to n. After gradients of all n workers are collected by the master node, they are used to update the model  (which is the synchronization point), and then the next iteration begins.
I know that through cpp part, I can somehow get "number of gradients accumulated" by master node, is there any possibility to achieve this from python interface? Since that will help a lot for my further process.

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to ping me to reopen if this is still an active issue. Thanks!
